pangeo_forge/cli.py

```diff
@@ -1,12 +1,47 @@
+import runpy
+
 import click
 
 
-@click.command()
+@click.group()
 @click.version_option()
 def main():
-    """Pangeo-Forge: A tool for building and publishing analysis ready datasets.
+    pass
+
+
+@click.command()
+@click.argument("pipeline", type=click.Path(exists=True))
+def lint(pipeline):
```

Comment:
```suggestion
def check(pipeline):
```

I feel like "lint" is a very developer specific term. We want to make this tool as accessible as possible to data managers.